bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -322,10 +313,13 @@ with h5py.File(outfilename, "w") as h5out:
     # write injections from recarray
     for name, injlist in (("found", found), ("missed", missed)):
         grp = h5out.create_group(name)
-        injs = injarr[injlist]
+        injs = allinjections[injlist]
         for field in injs.fieldnames:
-            grp[field] = injs[field]
-
+            # h5py does not support unicode strings
+            if 'U' not in str(injs[field].dtype):
+                grp[field] = injs[field]
+            else:
+                grp[field] = str(injs[field])
```

Comment:
I had to do this because h5py does not seem to support Unicode data types. It works, but there may be a better way to work around it.

Reply:
I get the h5py and Unicode issue. I think my issue with the last line is the following. `injs[field]` would be a Numpy array of strings here. What you want to do is convert the Numpy array of strings to a list of Python strings, which h5py will support. However, the last line will not do that. It will create a *single* string, containing just how Numpy would print the array. This should do what you want:
```suggestion
                grp[field] = [str(s) for s in injs[field]]
```